### PR TITLE
Fix Vulnerability - Ready

### PR DESCRIPTION
**Pull request by deleonio**

### CODE REVIEW

**Comment by martinoppitz** [comment]
I fixed the last pr check - https://github.com/martinoppitz/pathval/commit/e0983b66070f093b521a563115a79b55e0bc5587

**Comment by martinoppitz** [comment]
@vesln Please merge this PR

**Review by morozr — APPROVED** [review]

**Review by morozr — APPROVED** [review]

**Comment by deleonio** [comment]
Please merge it: see https://snyk.io/test/npm/@leanup/cli/1.0.74

**Comment by Ydot19** [comment]
Bump for visibility

File: test/.eslintrc [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "mocha": true
   },
   "rules": {
+    "eslint no-process-env": "off",
```
Comment (keithamus) [inline test/.eslintrc:10]:
```suggestion
    "no-process-env": "off",
```

**Review by keithamus — APPROVED** [review]

**Comment by keithamus** [comment]
Released as `v1.1.1`

**Comment by JBallin** [comment]
To update `chai`'s `pathval` dependency in your app, you can now run `npm i  -D chai`.

**Pull request merged**